packages/@aws-cdk/aws-synthetics/test/metric.test.ts

```diff
@@ -31,7 +31,7 @@ test('.metricXxx() methods can be used to obtain Metrics for the canary', () =>
     dimensions: { CanaryName: canary.canaryName },
     namespace: 'CloudWatchSynthetics',
     metricName: 'Failed',
-    statistic: 'Average',
+    statistic: 'Sum',
```

Comment:
Isn't this a breaking change?

Reply:
Somewhat yes, but also somewhat no in the sense that every bugfix is a breaking change and we generally don't call them out like that.

I guess that's worth a separate discussion in a separate PR though.